src/main/java/com/diffblue/javademo/UserAccess.java

```diff
@@ -3,6 +3,8 @@
 import com.diffblue.javademo.serveraccess.DatabaseDao;
 import org.bson.Document;
 
+// Copyright 2016-2018 Diffblue limited. All rights reserved.
```

Comment:
Putting the copyright after the imports seems strange to me - I'd probably put it right at the top, before even the package statement, but other files seem to have package, copyright, imports, e.g. src/main/java/com/diffblue/javademo/nestedobjects/User.java